shap/plots/_violin.py

```diff
@@ -37,48 +37,64 @@ def violin(
     color_bar=True,
     plot_size="auto",
     layered_violin_max_num_bins=20,
-    class_names=None,
```

Comment:
why did you remove these? That would make this a breaking change which we try to avoid

Reply:
Hi. Thanks for the code review. This argument never gets used in https://github.com/shap/shap/blob/master/shap/plots/_violin.py and I think it's safe to remove it.  @CloseChoice